tar.go

```diff
@@ -0,0 +1,125 @@
+package testcontainers
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func isDir(path string) (bool, error) {
```

Comment:
should be moved to `file.go` or something like that? IMO, tar.go is not the place for this. WDYT?

Reply:
Indeed. Will rename it asap